build.gradle

```diff
@@ -1,8 +1,8 @@
 plugins {
     id 'signing'
-    id 'fabric-loom' version '1.0-SNAPSHOT'
+    id 'fabric-loom' version '1.0.+'
```

Comment:
I would prefer this stayed as `SNAPSHOT`, the `+` syntax isn't in the maven specification to my knowledge.

Reply:
Well according to gradle, this is actually a valid option as it now compiles the entire thing using "1.0." at beginning but picks the highest number that the maven's website has.

Reply:
The problem is some older maven repositories might break with this. Hence I don't use it because I would consider it a bad practice at least to my knowledge.
[_The Maven version order algorithm is not compatible with Semantic Versioning 2.0.0. In particular, Maven does not special case the plus sign or consider build identifiers._](https://maven.apache.org/pom.html#version-order-specification)

Reply:
Will undo it then.